lib/fathead/bower/output.txt

```diff
@@ -0,0 +1,24 @@
+cache	A										<section class="prog__container">Manage bower cache<pre><code>\n$ bower cache <command> [<args>]\n</code></pre></section>	https://bower.io/docs/api/#cache
```

Comment:
@moollaza maybe adding 'command' to the article title (eg.`bower help command`) ? 
`bower home` may also benefit from getting the word `command` appended, as it also relates to bower homepage

In any case, it will show the IA and all the relevant results after, so for instance, for `bower help` one would still get the help resources results